Kickstarter-iOS/Views/Cells/LiveStreamContainerMoreMenuIconTextCell.swift

```diff
@@ -0,0 +1,70 @@
+import Library
+import LiveStream
+import Prelude
+
+internal final class LiveStreamContainerMoreMenuIconTextCell: UITableViewCell, ValueCell {
+
+  @IBOutlet private weak var contentStackView: UIStackView!
+  @IBOutlet private weak var iconImageView: UIImageView!
+  @IBOutlet private weak var rootStackView: UIStackView!
+  @IBOutlet private weak var subtitleLabel: UILabel!
+  @IBOutlet private weak var titleLabel: UILabel!
+
+  internal func configureWith(value moreMenuItem: LiveStreamContainerMoreMenuItem) {
+    switch moreMenuItem {
+    case .hideChat(let hidden):
+      self.iconImageView.image = UIImage(named: "speech-icon")
+      self.titleLabel.isHidden = true
+      self.subtitleLabel.text = hidden
+        ? localizedString(key: "Show_chat", defaultValue: "Show chat")
+        : localizedString(key: "Hide_chat", defaultValue: "Hide chat")
+    case .share:
+      self.iconImageView.image = UIImage(named: "share-icon")
+      self.titleLabel.isHidden = true
+      self.subtitleLabel.text = localizedString(key: "Share_live_stream",
+                                                defaultValue: "Share live stream")
+    default:
+      break
```

Comment:
yeah, if `.cancel` happening means something really bad happened then an assertion failure would be good to make it complain loudly